lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -10,9 +10,10 @@ def name
       # need: params['percentage']
       def is_enabled?(params = {}, context = nil)
         return false unless params.is_a?(Hash)
-        return false unless context.instance_of?(Unleash::Context)
 
         stickiness = params.fetch('stickiness', 'default')
+        return false unless context.instance_of?(Unleash::Context) || ['random', 'default'].include?(stickiness)
```

Comment:
This is a bit complex logic. Any chance to try to extract the check into a method, and use the method name to document what the check is trying to accomplish?